__init__.py

```diff
@@ -435,9 +464,9 @@ def handle_is_it_foggy(self, message: Message):
     def handle_is_it_raining(self, message: Message):
         """Handler for weather requests such as: is it raining today?
 
-        Args:
-            message: Message Bus event information from the intent parser
-0]       """
+                Args:
+                    message: Message Bus event information from the intent parser
+        0]"""
```

Comment:
some random text that's been left in from some point in history

Reply:
probably some sort of cut and paste error.  this has been fixed.